deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -35,7 +35,7 @@ data:
         meta:
           package:
             name: kubernetes
-            version: 1.9.0
+            version: 1.29.2
```

Comment:
Are we supposed to update this every time we have a new version of the integration?

Reply:
it would be great to know how you generated this manifest for next time

Reply:
For now I went to UI and created the standalone version of manifest from there. This included latest references for packages and manually I did the changes.

Reply:
> Are we supposed to update this every time we have a new version of the integration?

Good one, I don think we can follow the system integration changes all the time. I think we should try to keep updating yes but still trying to figure out how can we constantly. Maybe periodically once per quarter? Or from now on if an important k8s integration is added to have it in mind?